imports/ui/clients/viewAllClient.js

```diff
@@ -0,0 +1,57 @@
+import { Mongo } from 'meteor/mongo';
+//const SearchClient = Meteor.isClient ? new Mongo.Collection('search') : undefined;
+const SearchClient = new Mongo.Collection('search');
```

Comment:
??? why are you creating mongo collection here? pass search results via router data